python/src/LeastSquaresEquationsSolver_doc.i.in

```diff
@@ -0,0 +1,47 @@
+%feature("docstring") OT::LeastSquaresEquationsSolver
+"Least Square Solver (LeastSquaresEquationsSolver) for a system of non-linear equations.
+
+
+Parameters
+----------
+absError : positive float
+    Absolute error: distance between two successive iterates at the end point.
+    Default is :math:`10^{-5}`.
+relError : positive float
+    Relative error: distance between the two last successive iterates with
+    regards to the last iterate. Default is :math:`10^{-5}`.
+resError : positive float
+    Residual error: threshold for accepting a given solution. It is also used as residual error for the least square optimizer.
+    value. Default is :math:`0`.
+maximumFunctionEvaluation : int
+    The maximum number of evaluations of the function. Default is :math:`100`.
```

Comment:
Please introduce the associated ResourceMap key here too.

Reply:
Already existing. It is a bad copy/paste from `Brent` doc. This will be improved for this class